pages/linux/snap.md

```diff
@@ -0,0 +1,24 @@
+# snap
+
+> Tool for managing the *snap* self-contained softwawre packages.
+> Similar to what *apt* is for *.deb*.
+
+- Look for an app:
+
+`snap find {{package}}`
+
+- Install software:
+
+`snap install {{package}}`
+
+- Find installed *snap* software:
```

Comment:
As my previous suggestion about synonyms for actions, I would use `List` instead of `Find` here.